schema/rdl_schema_0.1.json

```diff
@@ -17,16 +16,21 @@
   ],
   "properties": {
     "title": {
+      "title": "Dataset title",
       "type": "string",
-      "description": "Title of the dataset"
+      "description": "The title of the dataset."
```

Comment:
Does "The Title of the dataset" in `description` not duplicate `title`?

Reply:
yes, I've adjusted to make them less duplicates